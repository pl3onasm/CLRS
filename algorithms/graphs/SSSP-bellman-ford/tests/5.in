s

s a 3
s c 5
s e 2
a b -4
b g 4
c d 6
d c -3
d g 8
e f 3
f g 7
h i 2
i j 3
j h -8

