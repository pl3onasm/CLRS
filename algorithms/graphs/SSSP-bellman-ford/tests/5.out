
--------------------
 Graph
 Directed
 Weighted
 Vertices: 11
 Edges: 13
--------------------
  a[1]: b (-4)
  b[1]: g (4)
  c[1]: d (6)
  d[2]: g (8), c (-3)
  e[1]: f (3)
  f[1]: g (7)
  g[0]
  h[1]: i (2)
  i[1]: j (3)
  j[1]: h (-8)
  s[3]: e (2), c (5), a (3)
--------------------


Shortest paths
Source: s
---------------------------------
Vertex: Parent, Distance from src
---------------------------------
  g: b, 3.00
  h: NIL, INF
  c: s, 5.00
  f: e, 5.00
  b: a, -1.00
  e: s, 2.00
  a: s, 3.00
  j: NIL, INF
  s: NIL, 0.00
  d: c, 11.00
  i: NIL, INF
---------------------------------
